Assets/EnemyHealthBar.cs

```diff
@@ -0,0 +1,28 @@
+using Unity.VisualScripting;
+using UnityEngine;
+using UnityEngine.UI;
+public class EnemyHealthBar : MonoBehaviour
+{
+    public Slider healthSlider;
+    private float maxHealth;
```

Comment:
is there a reason to give the healthbar it's own Max and Currenthealth rather than referencing the enemy's max and current health?
although not sure the enemy has a maxHealth itself. But it does keep track of the health for you already.

Reply:
yeah this code was useless, I made it for testing and now its removed